R/downloadbuttons.R

```diff
@@ -58,19 +58,20 @@ write_to_pdf_or_png <- function(file, type, renderFunction) {
   
 #' Create a download handler for raw data
 #' 
-#' @param filenameName of file to which to download
-#' @param continuousFile File to download if continuous
-#' @param binaryFile File to download if binary
+#' @param metaoutcome function to get th outcome to decide which file to download
+#' @param filename Name of file to which to download
+#' @param continuous_file File to download if continuous
+#' @param binary_file File to download if binary
 #' @return The created download handler
-create_raw_data_download_handler <- function(input, filename, continuousFile, binaryFile) {
+create_raw_data_download_handler <- function(metaoutcome, filename, continuous_file, binary_file) {
```

Comment:
I changed the method so that only the correct value was used, rather than passing the entire input through. It's just to reduce the scope of what the function can access.